doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -1086,6 +1086,12 @@ Controlling Typing Flags
    This :term:`boolean attribute` is similar to the :flag:`Universe Checking` flag, but on a per-declaration basis.
    Disable universe checking locally with ``bypass_check(universes)``.
 
+.. flag:: Cumulative Prop
+
+   This :term:`flag` can be used to control whether `Prop` is in
+   cumulativity with the predicative universes (`Set` and instantiations of `Type`).
+   It is on by default.
```

Comment:
```suggestion
   Controls whether `Prop` is cumulative for the predicative universes (`Set` and
   instantiations of `Type`).  It is on by default.
```

If correct.  "`Prop` is in cumulativity with the predicative universes" reads very strangely and unclear.